spec/row_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rspec'
+require '/lib/row'
+
+RSpec.describe Row do
+
+    xit "exists" do
+
+        row = Row.new
+        
+        expect(row).to be_an_instance_of(Row)
+    end
+
+    xit "has two readable attributes 'occupied' and 'player1' that toggleable true or false" do
+
+        row = Row.new
+
+        expect(row.occupied). to eq(FALSE)      #Initial state of attributes should be FALSE
```

Comment:
If row.occupied and row.player1 call separate methods, we should have two separate tests